app/src/renderer/vuex/modules/wallet.js

```diff
@@ -84,9 +85,9 @@ export default ({ commit, node }) => {
     },
     async queryWalletHistory({ state, commit, dispatch }) {
       commit("setHistoryLoading", true)
-      // let res = await node.coinTxs(state.address)
+      let res = await node.coinTxs(state.address)
```

Comment:
does history work again?!

Reply:
oh i uncommented this to see what `NiLiTx` looked like and it seemed to work with mock network. i'll remove tho since that's not the point of this pr